### PR TITLE
First pass at successful saving

### PR DESCRIPTION
**Pull request by tbaxter-18f**

### CODE REVIEW

**Comment by tbaxter-18f** [comment]
There is, honestly, a fair bit of hackery and half-bakedeness in here. Suggestions welcomed.

File: nrm_django/nrm_site/templates/admin/base.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <script src="{% static 'admin/js/nav_sidebar.js' %}" defer></script>
     {% endif %}
     <link href="{% static 'vendor/uswds-2.10.1/css/uswds.min.css' %}" rel="stylesheet" type="text/css" />
-    <link href="{% static 'css/site.css' %}" rel="stylesheet" type="text/css" />
+    <link href="{% static 'css/site.css' %}?v=8" rel="stylesheet" type="text/css" />
```
Comment (neilmb) [inline nrm_django/nrm_site/templates/admin/base.html:14]: Is this used for cache-busting? If so, do we want to change to a hash-based query parameter here?

File: nrm_django/grants/models.py [hunk 2]
```diff
@@ -414,7 +421,7 @@ class Grant(models.Model):
         max_length=34,
         blank=True,
         null=True,
-        help_text="Cooperators agreement number, if different then the Forest Service Agreement Number",
+        help_text="Cooperator's agreement number, if different then the Forest Service Agreement Number",
```
Comment (neilmb) [inline nrm_django/grants/models.py:424]: s/then/than/

File: nrm_django/grants/forms.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        instance = super().save(commit=False)
+        # first, we're gonna have to create our cn/pk
+        # * First character: Year
+        # * Second character: Region
+        # * Trailing characters: The instance created in
+        # * Middle characters: some unique ID
+        # TO-DO: figure out a better way to know the correct region. Maybe from contacts table?
+        year_char = datetime.date.today().strftime("%y")[1:]
+        short_uuid = str(uuid.uuid4())[:8]
+        instance_id = "10602"
```
Comment (neilmb) [inline nrm_django/grants/forms.py:53]: Could we comment or use a constant to convey that this is a magic value that happens to match lots of their current values?

File: nrm_django/grants/forms.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+from django import forms
+from django.core.exceptions import ValidationError
+
+from .models import Grant
+
+
+class GrantForm(forms.ModelForm):
+    class Meta:
+        model = Grant
```
Comment (neilmb) [inline nrm_django/grants/forms.py:12]: Love the way this makes creation easy to connect up with the existing list view.

**Review by neilmb — APPROVED** [review]
LGTM

File: nrm_django/nrm_site/templates/admin/base.html (hunk 1 above)
Comment (tbaxter-18f) [reply nrm_django/nrm_site/templates/admin/base.html:14]: It was. We don't _really_ need to keep it at all.

File: nrm_django/grants/forms.py (hunk 4 above)
Comment (tbaxter-18f) [reply nrm_django/grants/forms.py:12]: Modelforms + Generic views are just amazing.

File: nrm_django/grants/forms.py (hunk 3 above)
Comment (tbaxter-18f) [reply nrm_django/grants/forms.py:55]: Yeah, I noted it a bit below but we can certainly make it more explicit. I also wonder if there's any real value in keeping the instance ID at all, but that's a conversation for another day.

File: nrm_django/grants/models.py (hunk 2 above)
Comment (tbaxter-18f) [reply nrm_django/grants/models.py]: Nice catch. I spotted the apostrophe but not then/than. Also, don't blindly cut/paste help text from the user guide.

**Pull request merged**